### PR TITLE
[ADD] sale_order_product_recommendation_pwa_cache: Glue Module

### PR DESCRIPTION
**Pull request by chienandalu**
- [ ] Depends on https://github.com/OCA/sale-workflow/pull/1365 `sale_pwa_cache`

cc @Tecnativa TT27734

### CODE REVIEW

File: sale_order_product_recommendation_pwa_cache/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Firstname Lastname <email.address@example.org> (optional company website url)
```
Comment (pedrobaeza) [inline sale_order_product_recommendation_pwa_cache/readme/CONTRIBUTORS.rst]: Change this

Comment (chienandalu) [reply sale_order_product_recommendation_pwa_cache/readme/CONTRIBUTORS.rst]: done

**Comment by rousseldenis** [comment]
@pedrobaeza @chienandalu  I'm wondering if the pwa mechanism couldn't be transversal to be applied in the whole instance to avoid too much glue modules like this ?

**Comment by chienandalu** [comment]
The thing is that now the approach is to have a minimum sales form and from there add just the needed elements for the salesmen to work properly (or having a custom one)

**Comment by pedrobaeza** [comment]
The main amount of work is in the base module (`web_pwa_cache`), but each area can require a module for:

- Data cache specification.
- Specific optimized views.

That's why we have some glue modules for optional things. You can't force everyone wanting to have sales to put the product recommendation or the secondary unit.

**Comment by chienandalu** [comment]
As #1365 is dismissed in favour of the latter v13 developement I think this can be as well (as it's migrated in that version as well)

**Pull request closed without merging**